catalog/sql/sql.go

```diff
@@ -147,6 +149,19 @@ type sqlIcebergNamespaceProps struct {
 	PropertyValue sql.NullString
 }
 
+type sqlIcebergView struct {
+	bun.BaseModel `bun:"table:iceberg_views"`
+
+	CatalogName              string `bun:",pk"`
+	ViewNamespace            string `bun:",pk"`
+	ViewName                 string `bun:",pk"`
+	ViewSQL                  string
+	SchemaJSON               string
+	MetadataLocation         sql.NullString
+	PreviousMetadataLocation sql.NullString
+	Properties               map[string]string `bun:",json"`
```

Comment:
is this the same schema used by the JDBC/other sql based catalogs?

Reply:
Ah that makes sense. I have changed it accordingly :pray: